src/components/Profile/ProfileContainer.js

```diff
@@ -54,4 +46,13 @@ ProfileContainer.propTypes = {
   isAuth: PropTypes.bool,
 };
 
-export default connect(mapStateToProps, { getUserProfile })(WithUrlDataContainerComponent);
+const mapStateToProps = (state) => ({
+  profile: state.profilePage.profile,
+});
+
+
+export default compose(
+  connect(mapStateToProps, { getUserProfile }),
```

Comment:
Consider creating a variable for mapDispatchToProps, as you do for mapStateToProps.
Everywhere in the project, for the sake of consistency.